_data/harnesses/deserialization-errors/ksql.yml

```diff
@@ -1,3 +1,11 @@
+answer:
+  steps:
+    - title:
```

Comment:
Do we want a title for short answers? We have "short answers" that use the title section (Idempotent producer & Masking Data) and one that doesn't (Anomaly Detection). I personally don't love that the title text appears to be in a larger font size than "Short Answer" and find that a title can be redundant given the nature of this section. Thoughts @ybyzek?

Reply:
@awalther28

> I personally don't love that the title text appears to be in a larger font size than "Short Answer"

💯 agree, so the omission of a title was intentional.  

For clarity: are you saying you like this approach (as the PR Is now) and we should standardize on this for all KTs?

Reply:
@ybyzek glad to hear I'm not the only one to notice the font size difference. Thank you for looking into if we can change that :) I like this approach of omitting a title and would like to see future KT short answers do the same. 
I could be convinced to standardize adding a title if we can change the font size + the title provides value.

Reply:
My preference would be to have no title, so I'd take the PR as it stands now.  But I don't have a strong opinion, so I could go with a title (assuming we work out the font sizing)—just my 2 cents.

Reply:
One more advantage to having no title is that special characters can be added to `adoc` that can't be added to the title.  For example, `KSQL_PROCESSING_LOG` can have backticks, ONLY if it's in adoc, not title

```
`KSQL_PROCESSING_LOG`
```

So the advantages of no title are:

1. Easy, done, no more fiddling with font manipulation
2. Special characters.

Disadvantage:

1. Looks funny?

If we agree with no title, which I think we do...here's an audit:

```
aggregating-count/ksql.yml:answer:
aggregating-count/ksql.yml-  steps:
aggregating-count/ksql.yml-    - title: Persistent query using COUNT
--
aggregating-count/kstreams.yml:answer:
aggregating-count/kstreams.yml-  steps:
aggregating-count/kstreams.yml-    - title: count() method
--
anomaly-detection/ksql.yml:answer:
anomaly-detection/ksql.yml-  steps:
anomaly-detection/ksql.yml-    - title:
--
console-consumer-producer-basic/kafka.yml:answer:
console-consumer-producer-basic/kafka.yml-  steps:
console-consumer-producer-basic/kafka.yml-    - title: Console producer
--
deserialization-errors/ksql.yml:answer:
deserialization-errors/ksql.yml-  steps:
deserialization-errors/ksql.yml-    - title:
--
masking-data/ksql.yml:answer:
masking-data/ksql.yml-  steps:
masking-data/ksql.yml-    - title: Use the MASK ksqlDB function
--
message-ordering/kafka.yml:answer:
message-ordering/kafka.yml-  steps:
message-ordering/kafka.yml-    - title: Configure the producer for idempotency
```

So I could update this PR to fix _all_ the KTs that use titles, and move it to adocs.  @awalther28 @bbejeck  please let me know if you agree, and I'll jump on it ;)

Reply:
I agree 💯

Reply:
+1 for me